workflows/microbiome/README.md

```diff
@@ -0,0 +1,60 @@
+# Microbiome Workflows
+
+The following workflows can be used directly for microbiome data analysis, pathogen detection and tracking purposes. The workflows can be also adapted to any other sequencing technique. 
+
+To learn more about the following workflows and try them with real datasets, please check our tutorial on the Galaxy Training Network [GTN](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
+
+## Nanopore _Preprocessing_
+
+Before starting any analysis, it is always a good idea to assess the quality of your input data and to discard poor quality base content by trimming and filtering reads.
+
+Generally, we are not interested in the host sequences, rather only those originating from the pathogen itself. It is important to get rid of all host sequences and to only retain sequences that might include a pathogen, both in order to speed up further steps and to avoid host sequences compromising the analysis.
+
+### Input Datasets
+
+- Collection of sequenced Nanopore reads of all samples to be analized in a `fastq or fastq.gz` formate
+
+### Output Datasets
+
+- Collection of Pre-Processed Sequenced reads of all samples, ready for further analysis with the other workflows, in a `fastq or fastq.gz` formate
+
+- Tables indicating total number of reads before and after host sequences trimming, and the host sequences percentages found in each sample, these tables are needed for visualistion performed in **PathoGFAIR Samples Aggregation and Visualisation** workflow.
+
+## _Taxonomy Profiling_ and Visualisation with Krona
+
+In this workflow, we identify the different organisms found in our samples by assigning taxonomy levels to the reads starting from the kingdom level down to the species level and visualise the result.
+
+It’s important to check what might be the species of a possible pathogen to be found, it gets us closer to the investigation as well as discovering possible multiple pathogenetic infections if any existed.
+
+For taxonomy profiling Kraken2 tool is used along with one of its standard databases available on Galaxy, you can freely choose between Kraken2 different databases based on your input datasets. For visualisation multiple tools can be used, Krona pie chart (as default in this workflow), Phinch interactive tool, Pavian, etc.
+
+## _Gene-based Pathogen Identification_
```

Comment:
Is it specific for Nanopore data here?

Reply:
nope its general

Reply:
Preprocessing for Nanopore, also mention which type of data (metagenomics)